openstack/sap-seeds/templates/_flavors_deleted.tpl

```diff
@@ -410,6 +410,11 @@
     trait:CUSTOM_NUMASIZE_C48_M1459: required
     hw:cpu_cores: '24'  # cores-per-socket
     vmware:hw_version: vmx-18
+    {{- if .Values.hana_flavors_quota_separate }}
+    "quota:instance_only": "true"
+    "quota:separate": "true"
+    "limes:allow_growth": "false" # never report capacity above the current usage
```

Comment:
why not `quota:allow_growth`? i'd like to prevent section creep. "limes" is just a name, not a type/topic.

Reply:
`quota:allow_growth` looks like something defined by Nova. I want to be very specific about this being a Limes-specific thing.